### PR TITLE
add aliases from DSDA-Doom, fix docgen warnings

### PR DESCRIPTION
**Pull request by rfomin**

### CODE REVIEW

**Comment by fabiangreffrath** [comment]
Thank you! This is the pragmatic approach, and I like it and we sure can merge it. However, I was thinking about something like this in `d_main.c`:

```c
static const struct {
    const char *from;
    const char *to[3];
} cli_aliases[] = {
    {"-nomo", {"-nomonsters", NULL}},
    {"-uv",   {"-skill", "4", NULL}},
};
```

This is the more elegant solution. However, it comes with two issues. First, it doesn't register the aliased parameters with docgen. Second, it would be quite some code shuffle to inject a two-part argument like `-skill 4` intead of `-uv` into `argv[]`. The main advantage, though, would be that we could have aliases like `--version` and ` -version` all gathered in a central place.

**Comment by rfomin** [comment]
> The main advantage, though, would be that we could have aliases like `--version` and `-version` all gathered in a central place.

We have only two such aliases, `-version`/`--version` and `-help`/`--help`. We do not support `--` in the rest of the parms checking code. There are also old aliases from Boom in the style of this PR:`-deh`/`-bex` and `-dehout`/`-bexout`.

Alternatively, we can remove `docgen` and concentrate all the parameters in one big table, with help strings and everything else.

File: src/params.h [hunk 1]
```diff
@@ -102,10 +106,14 @@ General options: \n\
   -iwad <file>   Specify an IWAD file to use.\n\
 \n\
 Game start options: \n\
+  -nm                 Alias for -skill 5.\n\
+  -nomo               Alias to -nomonsters.\n\
```
Comment (fabiangreffrath) [inline src/params.h:110]: "Alias for" or "Alias to"?

Comment (rfomin) [reply src/params.h:110]: I changed it to "Alias to" in https://github.com/fabiangreffrath/woof/pull/1678/commits/1afd91afb28b1c50ea5f9b0402c45f4443eae3ab

**Review by fabiangreffrath — APPROVED** [review]

**Pull request merged**